src/transaction.js

```diff
@@ -768,7 +768,13 @@ const transaction = {
         if (response.success) {
           resolve(response);
         } else {
-          reject(response.message);
+          let message = '';
+          if (response.return_code === 'stratum_timeout') {
```

Comment:
I know it's been used only here, but I think we should make it a constant.

Reply:
Done